apps/openmw/CMakeLists.txt

```diff
@@ -155,6 +155,8 @@ if (ANDROID)
         android
         log
         dl
+        ${FREETYPE_LIBRARY_REL}
+        ${FREETYPE_LIBRARY_DBG}
```

Comment:
This shouldn't be needed. The freetype libraries should be contained in MYGUI_LIBRARIES.